o.n.core/src/org/netbeans/core/ui/ProductInformationPanel.java

```diff
@@ -94,8 +98,30 @@
     })
     public ProductInformationPanel() {
         initComponents();
+        descripTemplate = NbBundle.getMessage(ProductInformationPanel.class,
+                "LBL_description");
```

Comment:
`getMessage` takes parameters - pass all the `(String bundleString, String productVersion, String javaVersion, String vmVersion, String os, String encoding, String locale, String userDir, String cacheDir, String updates, int fontSize, String javaRuntime)` in and let the one who does the branding format that somehow.

Reply:
Enhancing branding this way is is also an API change, so you can still document it in `arch.xml` (I don't think there is `category="branding"`, if not, use `"property"`) and record it in `apichanges.xml` files + bump the version of the module.